### PR TITLE
[WPF] Fixed issue where reported dimensions mistakenly included window chrome

### LINKED ISSUE #965: Window height is greater than window frame
It seems that `Dimensions.get('window').height` is greater than the window frame. I think the height might include the window status bar's. For example, the blue area height is the same as the window height, which is get by `Dimensions.get('window').height` but when resizing window vertically, there's more blue area appears.

blue

### PR DESCRIPTION
**Pull request by kevinvangelder**
Closes #965.

Unfortunately this solution is pretty brittle but seems to be the best solution the interwebz have to offer.

### CODE REVIEW

**Comment by pre10der89** [comment]
Does the RenderSize or ActualHeight/ActualWidth properties give you the dimensions you are looking for?  You may be able to check those on the window itself or window. Content...

**Comment by kevinvangelder** [comment]
ActualHeight and ActualWidth still include the chrome. I didn't see RenderSize in the documentation but if it did what we want people would have been putting that as the answer instead of this hack on all the StackOverflow questions.

**Comment by pre10der89** [comment]
@kevin.vangelder I looked into your fix and it seems to be the correct math... the size calculated on size changed matches the actual height and width of the window's root content... However, the math is off if the WindowStyle is changed to None, which doesn't have a caption... this led me to wonder whether we should be calculating the dimensions based on the windows content directly instead; right now we have to do this math on every size changed event...  we pass the window into the ui manager module and down cast it to a framework element, which causes us to lose access to a lot of functionality on the window including access to the Content property... If we didn't do the down cast then we could get the dimensions by calling ActualWidth/ ActualHeight on _window.Content... This all assumes that ui manager module is designed to be the manager of the main window or the root view itself... if it makes sense to manage other aspects of the main window in the ui manner module (e.g. setting title text or setting the size of the window itself) then perhaps we should change the input argument type to Window... @rozele might have some input

**Comment by kevinvangelder** [comment]
@pre10der89 I think you are probably correct. I'll play around with that possibility and see if I can come up with a more robust solution.

**Comment by kevinvangelder** [comment]
It took a lot of trial and error, but I did come up with a simpler solution that should be less brittle and still accurate.

File: ReactWindows/ReactNative.Net46/UIManager/UIManagerModule.cs [hunk 1]
```diff
@@ -508,17 +508,17 @@ private void OnBoundsChanged(object sender, SizeChangedEventArgs args)
...
+
             return new Dictionary<string, object>
             {
                 {
                     "window",
                     new Dictionary<string, object>
                     {
-                        { "width", bounds.Width },
-                        { "height", bounds.Height },
+                        { "width", content?.ActualWidth ?? new double() },
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/UIManager/UIManagerModule.cs]: >new double()  [](start = 59, length = 13)

it's not clear what new double() does... can you just default to 0.0?

File: ReactWindows/ReactNative.Net46/UIManager/UIManagerModule.cs [hunk 2]
```diff
@@ -508,17 +508,17 @@ private void OnBoundsChanged(object sender, SizeChangedEventArgs args)
 
         private static IDictionary<string, object> GetDimensions()
         {
-            var bounds = new Rect() {Height = _window.Height, Width = _window.Width};
+            FrameworkElement content = (FrameworkElement)_window.Content;
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/UIManager/UIManagerModule.cs]: >FrameworkElement [](start = 12, length = 16)

var

**Comment by kevinvangelder** [comment]
@rozele updated.

**Comment by rozele** [comment]
:shipit:

**Pull request merged**